cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +32,27 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(wrongUser.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    signInPage.verifyEmailErorr();
```

Comment:
There seems to be a typo in the method name `verifyEmailErorr`. It should likely be `verifyEmailError` to correctly reflect the purpose of verifying an email error.